src/odb/include/odb/db.h

```diff
@@ -7642,22 +7659,110 @@ class dbPowerSwitch : public dbObject
 class dbScanChain : public dbObject
 {
  public:
+  dbSet<dbScanPartition> getScanPartitions() const;
+
+  dbSet<dbScanList> getScanLists() const;
+
+  // User Code Begin dbScanChain
+  const std::string& getName() const;
+
+  void setName(std::string_view name);
+
+  void setScanIn(dbBTerm* scan_in);
+  void setScanIn(dbITerm* scan_in);
+  std::variant<dbBTerm*, dbITerm*> getScanIn() const;
+
+  void setScanOut(dbBTerm* scan_out);
+  void setScanOut(dbITerm* scan_out);
+  std::variant<dbBTerm*, dbITerm*> getScanOut() const;
+
+  void setScanEnable(dbBTerm* scan_enable);
+  void setScanEnable(dbITerm* scan_enable);
```

Comment:
What does it mean for an iterm to be the scan_enable for a whole chain?  Does it imply some sort of chained connection?

Reply:
Is the scan enable pin that puts the scan chain into shifting mode. We need this pin for simulations and well, to let the ATE (Automatic Test Equipment) what pin needs to be set to 1 for shifting.

Reply:
When would that pin be an iterm and not a bterm?  iterms are internal to the block.

Reply:
Maybe I have a misconception of what an iterm is. Is this correct?

If I have a cell (let's say a mux) and I want to take the output pin of that mux, would that be an iterm?

If I have a top-level input port (let's say, in verilog, "module (my_port)", would that be a bterm?

Thanks!

Reply:
That's correct, in iterm blocks to an instance